docs/src/pages/components/avatars/VariantAvatars.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { deepOrange, green } from '@material-ui/core/colors';
+import Avatar from '@material-ui/core/Avatar';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import Grid from '@material-ui/core/Grid';
+
+const useStyles = makeStyles({
+  avatar: {
+    margin: 12,
+  },
+  orangeAvatar: {
+    margin: 12,
+    color: '#fff',
+    backgroundColor: deepOrange[500],
+  },
+  greenAvatar: {
+    margin: 12,
+    color: '#fff',
+    backgroundColor: green[500],
+  },
+});
+
+export default function VariantAvatars() {
+  const classes = useStyles();
+
+  return (
+    <Grid container justify="center" alignItems="center">
+      <Avatar
+        variant="circle"
```

Comment:
No need for the variant prop, circle is the default. In fact, no need for the circular demo - it's amply demonstrate throughout the rest of the page. (Smaller demos.)

Reply:
👍

Reply:
Much better!
![Capture d’écran 2019-11-03 à 13 55 11](https://user-images.githubusercontent.com/3165635/68085452-9228fd80-fe41-11e9-847d-3a8696b0852a.png)